src/trainer/DataView.js

```diff
@@ -1,45 +1,35 @@
 import React from 'react';
```

Comment:
We should also standardize the Dataview with the ones that exist in preprocessing / annotator pages

Reply:
I agree. At the moment, they are too different (bucket, MIP thickness, etc.) I've added an issue #28